include/cantera/oneD/StFlow.h

```diff
@@ -301,7 +321,12 @@ class StFlow : public Domain1D
     doublereal lambda(const doublereal* x, size_t j) const {
         return x[index(c_offset_L, j)];
     }
-
+	///////////////////////////////////////////////////////////
+	// Alex: here is the extra variable for the extra equation//
```

Comment:
Please add useful comments. Extra variables in extra equations won't help anyone else in the future.